test/partisan_SUITE.erl

```diff
@@ -38,7 +38,7 @@
 -include_lib("kernel/include/inet.hrl").
 
 -define(APP, partisan).
--define(CT_SLAVES, [rita, sue, bob, jerome]).
+-define(CLIENT_NUMBER, 3).
```

Comment:
Yes, but you need to increase the waiting time to make the cluster stable. If test checks the symmetric status in the middle of exchanging join messages, it will fail.